fs/zbd_zenfs.h

```diff
@@ -151,8 +167,9 @@ class ZonedBlockDevice {
 
   std::shared_ptr<BackgroundWorker> meta_worker_;
   std::shared_ptr<BackgroundWorker> data_worker_;
-  std::list<Zone *> active_zones_list_;
-  std::mutex active_zone_list_mtx_;
+  std::vector<Zone *> active_zones_vec_;
```

Comment:
Could you also add a comment here to explain what the `active_zones_vec_` does? I think it would be much easier for new developers to jump in.

Reply:
Yeah, pretty much comments need to add.